src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -13,97 +12,40 @@
-    portMotor.config_kD(Constants.DrivetrainPID.kPIDLoopIdx, Constants.DrivetrainPID.kGains.kD, Constants.DrivetrainPID.kTimeoutMs);
-    starboardMotor.config_kF(Constants.DrivetrainPID.kPIDLoopIdx, Constants.DrivetrainPID.kGains.kF, Constants.DrivetrainPID.kTimeoutMs);
-		starboardMotor.config_kP(Constants.DrivetrainPID.kPIDLoopIdx, Constants.DrivetrainPID.kGains.kP, Constants.DrivetrainPID.kTimeoutMs);
-		starboardMotor.config_kI(Constants.DrivetrainPID.kPIDLoopIdx, Constants.DrivetrainPID.kGains.kI, Constants.DrivetrainPID.kTimeoutMs);
-    starboardMotor.config_kD(Constants.DrivetrainPID.kPIDLoopIdx, Constants.DrivetrainPID.kGains.kD, Constants.DrivetrainPID.kTimeoutMs);
-    
-    // Get the current abs position, and set the rel sensor to match
-    int absolutePortPosition = portMotor.getSensorCollection().getPulseWidthPosition();
-    int absoluteStarboardPosition = starboardMotor.getSensorCollection().getPulseWidthPosition();
-
-		/* Mask out overflows, keep bottom 12 bits */ // From CTRE Examples
-		absolutePortPosition &= 0xFFF;
-		if (Constants.DrivetrainPID.portSensorPhase) { absolutePortPosition *= -1; } // Invert Once
-    if (Constants.DrivetrainPID.portMotorInvert) { absolutePortPosition *= -1; } // Invert Twice
-    
-    absoluteStarboardPosition &= 0xFFF;
-		if (Constants.DrivetrainPID.portSensorPhase) { absoluteStarboardPosition *= -1; } // Invert Once
-		if (Constants.DrivetrainPID.starboardMotorInvert) { absolutePortPosition *= -1; } // Invert Twice
-		
-    /* Set the quadrature (relative) sensor to match absolute */ // From CTRE Examples
-    portMotor.setSelectedSensorPosition(absolutePortPosition, Constants.DrivetrainPID.kPIDLoopIdx, Constants.DrivetrainPID.kTimeoutMs);
-    starboardMotor.setSelectedSensorPosition(absoluteStarboardPosition, Constants.DrivetrainPID.kPIDLoopIdx, Constants.DrivetrainPID.kTimeoutMs);
-    
+    // Set motors to default on boot
+    portMotor.restoreFactoryDefaults();
+    portMotorSlave.restoreFactoryDefaults();
+    starboardMotor.restoreFactoryDefaults();
+    starboardMotorSlave.restoreFactoryDefaults();
 
     // Set Inverted
-    portMotor.setInverted(Constants.DrivetrainPID.portMotorInvert); // Sets the output of the motor backwards
-    portMotorSlave0.setInverted(Constants.DrivetrainPID.portMotorInvert);
-    portMotorSlave1.setInverted(Constants.DrivetrainPID.portMotorInvert);
-
-    starboardMotor.setInverted(Constants.DrivetrainPID.starboardMotorInvert);
-    starboardMotorSlave0.setInverted(Constants.DrivetrainPID.starboardMotorInvert);
-    starboardMotorSlave1.setInverted(Constants.DrivetrainPID.starboardMotorInvert);
+    portMotor.setInverted(false);
+    starboardMotor.setInverted(true);
+    //portMotor.setInverted(Constants.DrivetrainPID.portMotorInvert); // Sets the output of the motor backwards
+    //starboardMotor.setInverted(Constants.DrivetrainPID.starboardMotorInvert);
 
```

Comment:
We should change the inversion point of the drivetrain to one place. There's also inversion in HumanControl.java

Reply:
Good point, ill make a new branch for `driver-cleanup` to fix this in the future.